apps/regression/model/linear_model.cpp

```diff
@@ -28,7 +27,7 @@ double LinearModel::levelSet(double * modelCoefficients, double xMin, double ste
   if (a == 0) {
     return NAN;
   }
-  return (y-b)/a;
+  return y-b;
```

Comment:
There is no b coefficient in the linear model, so the line `double b = modelCoefficients[1];` should be removed.
The levelSet is supposed to give x given y, so here it should return y/a

Reply:
A few lines below, in fit, please remove `modelCoefficients[1] = store->yIntercept(series);`

Reply:
And again in `LinearModel::partialDerivate`, the case
` if (derivateCoefficientIndex == 1) {
    // Derivate: 1;
    return 1;
  }`
should not exist